utils/ascii_terminal_colors.py

```diff
@@ -0,0 +1,10 @@
+class ASCIITerminalColors:
+    HEADER = '\033[95m'
+    OKBLUE = '\033[94m'
+    OKCYAN = '\033[96m'
+    OKGREEN = '\033[92m'
+    WARNING = '\033[93m'
+    FAIL = '\033[91m'
+    ENDC = '\033[0m'
+    BOLD = '\033[1m'
+    UNDERLINE = '\033[4m'
```

Comment:
Use these to colour the text output of a `print` statement. See my changes to `__init__.py` as an example.